docs/migrate.rst

```diff
@@ -16,6 +16,30 @@ Pipelines from-sources-complete and from-sources-skip-build are removed
 
 The previously deprecated pipelines ``from-sources-complete`` and ``from-sources-skip-build`` have been removed. Specify ``--pipeline=from-sources`` instead.
 
+Rally requires a subcommand
+^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Previously a subcommand was optional when running a benchmark. With Rally 2.1.0 a subcommand is always required. So instead of invoking::
+
+    esrally --distribution-version=7.10.0
+
+Invoke Rally with the ``race`` subcommand instead::
+
+    esrally race --distribution-version=7.10.0
+
+
+Running without a track is deprecated
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Previously Rally has implicitly chosen the geonames track as default when ``--track`` was not provided. To ensure the workload is chosen deliberately, it is now deprecated to not specify a track. So instead of invoking::
```

Comment:
> To ensure the workload is chosen deliberately, 

What if we explain a bit more e.g.

> We want users to make a conscious choice of the workload and not specifying the track explicitly is deprecated (to be removed in a future version). So, instead of invoking::

Reply:
Thanks; I've pushed a slighty modified version of this in https://github.com/elastic/rally/pull/1175/commits/0e8a57081661817d1f1646469a76bf512e54b4af.